### DISCUSSION TITLE
Farms Domesticação Injustas e mal balanceadas

### DISCUSSION THREAD

**Discussion by Batatoones** [leogianfagna/IssueTracker#984] [Opinioes]
### Servidor afetado

Todos

### Descrição

Atualmente as farms de domesticar no vanillew estão quebrando o sistema de competição do servidor. Motivo de estarem quebrando são:
- Farm AFK;
- Compete em 3 torneios sem esforço algum sendo eles : domesticar + vigilante e experiência geral
- Injusto com quem começa no servidor 
- Monopoliza torneios e liga

Sugestão: Nerfar drásticamente exp ganho de mobs vindos de spawners e visar farmas de mobs spawnados naturalmente como era antigamente com pillagers(quem sabe).

**Comment by leogianfagna** [comment]
Já nerfei 95% da ultima vez, só se eu nerfar 99,9%. (ultimo nerf já levei xingo), tem certeza?

**Comment by Batatoones** [comment]
Mesmo você nerfando, ainda ta muito forte kkkkkkk é só tu ver que tem gente com 4k de domesticar
